_data/sitemap.yml

```diff
@@ -0,0 +1,8 @@
+- title: HOME
```

Comment:
A file used to indicate the sitemap of this website.
Used to find where currently we are